### PR TITLE
ICBM and Bank Transfer investment

### PR DESCRIPTION
**Pull request by trival**
Related: 
https://github.com/Neufund/platform-frontend/issues/1235
https://github.com/Neufund/platform-frontend/issues/1397

### CODE REVIEW

File: app/modules/investmentFlow/sagas.ts [hunk 1]
```diff
@@ -149,6 +182,79 @@ function* setGasPrice(): any {
...
+    // fill up etherToken with ether from walle}t
+  } else {
+    const ethVal = new BigNumber(i.ethValueUlps);
+    const difference = ethVal.sub(etherTokenBalance);
+    const txCall = contractsService.etherToken.depositAndTransferTx(etoId, ethVal, [""]);
+    txDetails = {
+      to: contractsService.etherToken.address,
+      from: selectEthereumAddressWithChecksum(state.web3),
+      input: txCall.getData(),
+      value: difference.round().toString(),
```
Comment (rudolfix) [inline app/modules/investmentFlow/sagas.ts]: if there's any need for rounding then there is something wrong much earlier. isn't `i.ethValueUlps` integer?

Comment (trival) [reply app/modules/investmentFlow/sagas.ts]: good point. In a prior version i used euro value and divided by etherPrice, that is why the round was still there, but it is unnecessary now.

File: app/modules/investmentFlow/sagas.ts [hunk 2]
```diff
@@ -88,7 +95,29 @@ function validateInvestment(state: IAppState): EInvestmentErrorState | undefined
...
+      ) > 0
+    ) {
+      return EInvestmentErrorState.ExceedsWalletBalance;
+    }
+  }
+
+  if (investmentFlow.investmentType === EInvestmentType.ICBMEth) {
+    if (
+      compareBigNumbers(
+        addBigNumbers([etherValue, gasPrice]),
```
[the remark this answers is not in the document]
Comment (trival) [reply app/modules/investmentFlow/sagas.ts]: 👍

File: app/modules/investmentFlow/sagas.ts [hunk 3]
```diff
@@ -88,7 +95,29 @@ function validateInvestment(state: IAppState): EInvestmentErrorState | undefined
...
+      ) > 0
+    ) {
+      return EInvestmentErrorState.ExceedsWalletBalance;
+    }
+  }
+
+  if (investmentFlow.investmentType === EInvestmentType.ICBMnEuro) {
+    if (
+      compareBigNumbers(
+        addBigNumbers([etherValue, gasPrice]),
```
[the remark this answers is not in the document]
Comment (trival) [reply app/modules/investmentFlow/sagas.ts]: 👍

File: app/components/modals/txSender/investment-flow/BankTransferDetails.tsx [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+    );
+  },
+);
+
+const BankTransferDetails = appConnect<IStateProps, IDispatchProps>({
+  stateToProps: state => {
+    return {
+      accountName: selectClientName(state.kyc),
+      country: selectClientCountry(state.kyc),
+      recipient: "Fifth Force GMBH",
```
Comment (rudolfix) [inline app/components/modals/txSender/investment-flow/BankTransferDetails.tsx]: can we take recipient, iban and bic from environment variables?

**Pull request merged**